integrations/jetpack/jetpack.php

```diff
@@ -76,7 +76,8 @@ public function jetpack_widget_get_top_posts( $posts ) {
 	 */
 	public function grunion_contact_form_field_html_filter( $r, $field_label ) {
 		if ( function_exists( 'icl_translate' ) ) {
-			if ( pll_current_language() !== pll_default_language() ) {
+			$curlang = pll_current_language();
+			if ( empty( $curlang ) || ! $curlang->is_default ) {
```

Comment:
I thought you reverted this in a previous PR.

Reply:
I missed that during the rebase ... Fixed in 8b750e43